### DISCUSSION TITLE
Organisation

### DISCUSSION THREAD

**Discussion by kolumnin** [community/community#28301] [GitHub Education]
I started with a personal account at GitHub, but when I came across Organisation, it is conflicting. Can the personal account be considered as an organisation account with the members limited to one? @kolumnin

**Comment by jfullstackdev** [comment]
you can convert that to Org for granular functionalities that are not present but you will be losing your personal account, just simply create another Org, 

as for me, I'm using Orgs to organize my content, even if there are no members, it's just me, 

also, it will not follow the personal root of GitHub Pages from your personal account, for example

https://github.com/devz-throwable

I own this Org, it's just like a separate user because the default domain when GitHub Pages is enabled does not even contain `jdevstatic`,

https://devz-throwable.github.io/
